packages/backend-test-utils/api-report.md

```diff
@@ -403,6 +403,13 @@ export type ServiceMock<TService> = {
 };
 
 // @public
+export function setupMswHandlers(worker: {
```

Comment:
I realized that what this is doing is registering Jest hooks for the MSW server or worker, although since we're always testing in Node.js it's always gonna be a server. Thinking that `registerMswTestHooks` might be a good name? "Test" instead of "Jest" because the `afterAll` etc. are commonly supported in other testing libraries too. Was also thinking that "register" is a pretty good verb to use for globally mutating actions, although "setup" is fine by me too.

I think the original name might just be a bit confusing, this really has nothing to do with request mock handlers 😅

Reply:
Makes sense, just renamed 🙏🏻